src/index.ts

```diff
@@ -1,3 +1,89 @@
-export default function helloWorld(): string {
-	return 'Hello world!';
-}
+// Imports
+import {
+	CSSValidator,
+	ValidateTextOptions,
+	ValidateTextResultBase,
+	ValidateTextResultWithWarnings,
+	W3CCSSValidatorResponse,
+} from './types';
+import * as https from 'https';
```

Comment:
Is this going to break in a browser? We probably need to update Travis CI to ensure it does some testing in a browser environment...

Reply:
Possibly... I did it for the intellisense and typings that come along with it, which you don't get from require. When it compiles down it is translated to a require, I don't know if that makes a difference.

Reply:
Let's see what happens once we get browser-based testing up-and-running.